cloudmarker/checks/firewallrulecheck.py

```diff
@@ -0,0 +1,119 @@
+"""Firewall Rule Check plugin for GCP Firewall rule record.
+
+This modules define a :class:`FirewallRuleCheck` that identifies weak Firewall
+rules for Google Cloud Platform.
+"""
+
```

Comment:
Please update the commit message to

```
Add FirewallCheck plugin
```
so that it matches the class name exactly and also that it is consistent with your previous commit message for GCPCloud plugin.